counterpoll/main.py

```diff
@@ -234,3 +235,48 @@ def show():
 
     click.echo(tabulate(data, headers=header, tablefmt="simple", missingval=""))
 
+def _update_config_db(state, filename):
+    """ Update counter configuration in config_db file """
+    with open(filename) as config_db_file:
+        config_db = json.load(config_db_file)
+
+    counters = [
```

Comment:
Is there any reason you are hardcoding this list here? Why not change the state for all counters in FLEX_COUNTER_TABLE?

Reply:
No. Updated and iterated over them. Thanks!